cmd/pktvisord/main.cpp

```diff
@@ -68,6 +69,10 @@ static const char USAGE[] =
     Configuration:
       --config FILE               Use specified YAML configuration to configure options, Taps, and Collection Policies
                                   Please see https://pktvisor.dev for more information
+    Crashpad:
```

Comment:
so we also need a way for users to turn this off, even in cases where they are using a container that hasn't enabled by default. perhaps `--cp-disable` and/or checking an environment variable

Reply:
Don't get why should we need to add this disable. if the crashpad configurations are no passed correctly, the crashpad code will not be executed

Reply:
If it's only going into the debug container, then that's okay. But if it's going to be enabled by default in the release container, then people need a way to shut it off.